zavod/zavod/meta/http.py

```diff
@@ -1,16 +1,21 @@
 from typing import Any, Dict, List
+
 from urllib3.util import Retry
 from banal import ensure_list
 from zavod import settings
 
+DEFAULT_RETRY_STATUS_CODES = [500, 502] + list(Retry.RETRY_AFTER_STATUS_CODES)
+
 
 class HTTP(object):
     def __init__(self, data: Dict[str, Any]) -> None:
-        self.total_retries: int = data.get("total_retries", 3)
-        self.backoff_factor: float = data.get("backoff_factor", 1)
-        statuses = ensure_list(
-            data.get("retry_statuses", list(Retry.RETRY_AFTER_STATUS_CODES))
+        self.total_retries: int = data.get("total_retries", settings.HTTP_RETRY_TOTAL)
+        self.backoff_factor: float = data.get(
+            "backoff_factor", settings.HTTP_RETRY_BACKOFF_FACTOR
         )
+        statuses = ensure_list(data.get("retry_statuses", DEFAULT_RETRY_STATUS_CODES))
+        self.backoff_max: int = settings.HTTP_RETRY_BACKOFF_MAX
```

Comment:
I'm a little nervous of being limited by this. Do you feel there's a risk of accidentally backing off too much?

If we do want it, we should probably document that there's a max (https://zavod.opensanctions.org/metadata/#http-options)

For context though, I think this limit isn't limiting yet (heh) - this is the longest that comes to mind https://github.com/opensanctions/nomenklatura/pull/186/files#diff-b93af8d376050f2a2530fac104d06349b73203c4d6b4a5c5ebd9ee244da8e925R86

Reply:
This is the default, see [the docs](https://urllib3.readthedocs.io/en/stable/reference/urllib3.util.html#urllib3.util.Retry) and [the code](https://github.com/urllib3/urllib3/blob/04662c9ae08c9f63fa254772d7618db65123a35e/src/urllib3/util/retry.py#L213). I now added a comment.

I considered introducing a None to keep this at the default, but we don't really have this pattern anywhere, so I went with this.

Reply:
interesting! Then it's nice that it's configurable while we're here :)